op-e2e/e2eutils/secrets.go

```diff
@@ -43,8 +45,6 @@ type MnemonicConfig struct {
 	Mallory string
 }
 
-// Secrets computes the private keys for all mnemonic paths,
```

Comment:
why are these doc comments getting removed?

Reply:
I coped this from Hive - can add it back.